### ISSUE TITLE
NVDA does not read selected items in the extended lists (combo boxes with a "multiple" property) when the list is collapsed (Firefox)

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#407]
**Reported by vtsaran on 2009-08-30 22:09**
NVDA does not render the names (values) of selected items in the extended lists into the virtual buffer (see attached example).

**Comment by leonardder** [comment]
@jcsteh: Would it be possible to get the former attachment from the trac archive back so it is possible to investigate whether this problem still exists?

**Comment by jcsteh** [comment]
[listtext.html.txt](https://github.com/nvaccess/nvda/files/1069973/listtext.html.txt) (rename to .html)

This issue still exists. It isn't specific to multi-select lists; NVDA doesn't render the selected item for single select lists either.

I was originally concerned about performance here because crawling all children to work out which ones are selected sounds like a performance killer. However, [IAccessible::accSelection](https://msdn.microsoft.com/en-us/library/windows/desktop/dd318486(v=vs.85).aspx) can return multiple children, though whether browsers actually fully implement this is an open question.

**Comment by fisher729** [comment]
@jcsteh @leonardder  I think this can be closed now. Using Firefox and NVDA 2019.1.1, I press Enter on the list, and can move to its three items.

**Comment by jcsteh** [comment]
It's true that this was partially fixed in #9166. However, I made a decision there to only render the first selected item, rather than all selected items. My concern with rendering all selected items is that there could be a lot of them. Strictly speaking, this issue asks for all values in a multiple select list box to be rendered. Personally, I don't think we should do this, but that's not really my decision. CC @MichaelDCurran for an authoritative opinion.

**Comment by Adriani90** [comment]
maybe @feerrenrut can propose further steps in regard to this issue considering @jcsteh's opinion above?

**Comment by Adriani90** [comment]
Actually I agree with the comment by Jamie above. I would say it would be enough to know the first selected element in browse mode when the list is colapsed along with the number of all other selected elements, like proposed in #5005. 
For example "list box volvo and 5 other elements selected".
When expanding the list and navigating in focus mode, one could navigate the selected items by ctrl+up and ctrl+down arrow.

Thus, I am closing this issue in favor of #5005 where this discussion can be continued.
